.github/workflows/set_labels.sh

```diff
@@ -27,26 +27,41 @@ MINIMAL="v: minimal"
 SMALL="v: small"
 MODERATE="v: moderate"
 LARGE="v: large"
+CI_MANAGER="p: CI Manager"
 
 DELETE_LABELS=("$MINIMAL" "$SMALL" "$MODERATE" "$LARGE")
 
 if [ "$CHANGES" -gt "$LARGE_THRESHOLD" ]; then
-    SIZE_LABEL="$LARGE"
+    LABELS="$LARGE"
 elif [ "$CHANGES" -gt "$MODERATE_THRESHOLD" ]; then
-    SIZE_LABEL="$MODERATE"
+    LABELS="$MODERATE"
 elif [ "$CHANGES" -gt "$SMALL_THRESHOLD" ]; then
-    SIZE_LABEL="$SMALL"
+    LABELS="$SMALL"
 else
-    SIZE_LABEL="$MINIMAL"
+    LABELS="$MINIMAL"
 fi
 
-DELETE_LABELS=("${DELETE_LABELS[@]//${SIZE_LABEL}/}")
+DELETE_LABELS=("${DELETE_LABELS[@]//${LABELS}/}")
 
 # API for adding labels on the Pull Request 
 API_URL="https://api.github.com/repos/$REPOSITORY/issues/$PR_NUMBER/labels"
 
-echo "Adding label: ${SIZE_LABEL[@]}"
-for LABEL in "${SIZE_LABEL[@]}"; do
+# 'CI Manager' label
+CHANGED_PATH=$(git diff --name-only $PR_BASE_SHA $PR_HEAD_SHA)
+CI_PATH=($CI_PATH)
+ci_label="false"
+for item in "${CI_PATH[@]}"; do
+    [[ "$CHANGED_PATH" == "${item}"* ]] && ci_label="true" && break
+done
```

Comment:
the intended logic is different: We want this label when _only_ changes in CI_PATH are made.